### ISSUE TITLE
Error message improvement (2)

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#140]
Use case: '`Maximum_version`' and '`Minimum_version`' not defined.

Outcome are the following two lines in debug log file:

```
Running without Robot Framework AIO version check!
Robot Framework AIO version check passed!
```

modified by Thomas:
No message shall be shown if both,  Minimum_version and Maximum_version is not  provided.

**Comment by HolQue** [comment]
Oh, just one moment.

Also the fact that no version control is executed, is an important information.

The first line should stay in this case:

`Running without Robot Framework AIO version check!`

**Comment by test-fullautomation** [comment]
Hi Son,

Holger's comment is O.K. for me. If both Minium_version and Maximum_version is not provided we log only:
`Running without Robot Framework AIO version check!`

Please implement accordingly.

Thank you,
 Thomas

**Comment by namsonx** [comment]
Hello Thomas, Hello Holger,

I create the pull-request #161 for this ticket.

Thank you, 
Son

**Comment by test-fullautomation** [comment]
Hi Son,
please check my findings for #161.
Thank you,
 Thomas

**Comment by test-fullautomation** [comment]
solved and merged with #161
